gettingStarted.md

```diff
@@ -42,8 +42,15 @@ sample implementation has been provided for
 
 [MySQL Java Broker](https://github.com/cloudfoundry-community/cf-mysql-java-broker):
 A Java port of the Ruby-based
+
 [MySQL broker](https://github.com/cloudfoundry/cf-mysql-broker) above.
 
+[Open Service Broker](https://github.com/swisscom/open-service-broker): 
```

Comment:
Could we add "Swisscom" to the name since "Open Service Broker" is a pretty generic name?

Reply:
+1 to this suggestion. I have heard confusion about the project name from others in the community.